integration-test/src/main/java/com/sequenceiq/it/cloudbreak/context/TestContext.java

```diff
@@ -336,7 +335,7 @@ T doAction(T entity, Class<? extends MicroserviceClient> clientClass, Action<T,
 
     protected <T extends CloudbreakTestDto, U extends MicroserviceClient>
     T doActionAsAdmin(T entity, Class<? extends MicroserviceClient> clientClass, Action<T, U> action, String who) throws Exception {
-        return action.action(getTestContext(), entity, getAdminMicroserviceClient(clientClass));
+        return action.action(getTestContext(), entity, getAdminMicroserviceClient(clientClass, Crn.fromString(entity.getCrn()).getAccountId()));
```

Comment:
Will this work with the old local setup for dp cli command? Since in that run there is no concept of userCrn.

Reply:
what do you mean by old local setup? i guess tests should run against same version of cbd, or is there some tests which uses older cbd than the version of integration-test.jar? here i am using the crn of the entity, not usercrn, so it should work. (side note: this method is not used, so probably i will delete it when i update this pr with secret versioning change)

Reply:
removed